install.sh

```diff
@@ -416,10 +416,10 @@ function install_liqo() {
 	${KUBECTL} create namespace "${LIQO_NAMESPACE}" 1>/dev/null 2>&1 || true
 	local LIQO_CHART="${TMPDIR}/${LIQO_CHARTS_PATH}"
 	info "[INSTALL]" "Packaging the chart..."
-	${HELM} package --version="${LIQO_CHART_VERSION}" --app-version="${LIQO_CHART_VERSION}" "${LIQO_CHART}" --dependency-update >/dev/null ||
+	${HELM} --debug package --version="${LIQO_CHART_VERSION}" --app-version="${LIQO_CHART_VERSION}" "${LIQO_CHART}" --dependency-update >/dev/null ||
```

Comment:
This script will be deprecated. Why do you need to add --debug?

Reply:
It was for me to help with debugging (and as a possible improvement to be cherry-picked in a PR), I'll drop the commit.